packages/react-magma-dom/src/components/Dropdown/DropdownExpandableMenuGroup.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import styled from '../../theme/styled';
+import { Accordion, AccordionProps } from '../Accordion';
+import { DropdownContext } from './Dropdown';
+
+const StyledAccordion = styled(Accordion)<{
+  testId?: string;
+}>`
+  border: none;
+`;
+
+export interface DropdownExpandableContext {
+  hasIcon?: boolean;
+  isExpandablePanel?: boolean;
+}
+
+export const DropdownExpandableContext =
+  React.createContext<DropdownExpandableContext>({});
+
+export const DropdownExpandableMenuGroup = React.forwardRef<
+  HTMLDivElement,
+  AccordionProps
+>(props => {
+  const { children, testId, ...other } = props;
+
+  const context = React.useContext(DropdownContext);
+
+  let hasIcon = false;
```

Comment:
I tend to prefer long variable names that are very clear over short ones that are less descriptive. In this case, the MenuGroup wouldn't have icons-- the children menu button is what we are after. Maybe we can call this `expandableMenuButtonHasIcon` or something like that.